### ISSUE TITLE
In the `code/` folder, many README.md files are outdated

### ISSUE THREAD

**Issue by balacij** [JacquesCarette/Drasil#2857]
As the title says, many of the README.md files our outdated.

Some files are completely outdated and discuss non-existent files, or files that moved elsewhere.

**Comment by daijingz** [comment]
Automatically Selected and started (Not with any permission, I just start working now)

**Comment by daijingz** [comment]
Update 1: In the `/Drasil/code/` repository; the README.md file forgot to explain the newly-added `drasil-codelang` folder. This is been updated on #3959

**Comment by smiths** [comment]
@balacij, I agree that the review process is the way to update our documentation.  I also agree that Haddock should be a higher priority than the `README` file.

Should we think about adding a PR template?  We could include in the template reminders to update the Haddock documentation.

**Comment by smiths** [comment]
I've never used a PR template before, either, but the issue templates have been handy, especially in capstone.

There are plenty of examples online.  There is a list at the bottom of this article on [Github pull request templates](GitHub pull request template).  I think we want something simple, at least to start with, like the first template example:

```
## Describe your changes and why they are necessary

## Issue ticket number and link

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have added Haddock documentation
- [ ] I have followed the Drasil naming conventions
- [ ] If this PR changes stable, I have run `make stabilize`
```

I've made a few modifications from the original source.  You probably have some additional ideas of things we could add to the checklist.  We can't add the point about following naming conventions until we've documented our naming conventions.  Do we want to suggest a local lint as part of the checklist?

**Comment by JacquesCarette** [comment]
Can we semi-Drasil-ify this? Maybe we could extract the `README.md` from the Haskell code itself, once we have the right 'template'?

**Comment by balacij** [comment]
I am surprised: https://pandoc.org/MANUAL.html#general-options . Pandoc supports Haddock as an input format, I'm not sure how well it will work. Maybe someone can look at it next summer?
